packages/gcp/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import * as ff from '@google-cloud/functions-framework';
+import {getCompositions} from '@remotion/renderer';
+import {getBrowserInstance} from './helpers/get-browser-instance';
+
+ff.http('TypescriptFunction', async (req: ff.Request, res: ff.Response) => {
+	const AWSUrl =
+		'https://remotionlambda-11kow3vq6f.s3.us-east-1.amazonaws.com/sites/xmycbufjs3/index.html';
+	const GCPUrl =
+		'https://storage.googleapis.com/remotionlambda-test/sites/xmycbufjs3/index.html';
+
+	// GCP isn't working, perhaps because because index.html references script files with a relative path, and it needs to be referencing these files with a full URL?
+	// At least the AWS url works
+	const comps = await getCompositions(AWSUrl);
+	console.log('comps', comps);
+
+	const browserInstance = await getBrowserInstance(true, {});
+	console.log('browserInstance', browserInstance);
```

Comment:
when running this locally, it fails on line 47 as it doesn't have Chromium

Reply:
Need to get it from the Lambda binaries and check it in, also the path is hardcoded in Lambda to `/opt/bin/chromium` I think

Reply:
So it should work if the browser is being put in the right path, or the path is adjusted for GCP